openpype/tools/publisher/control.py

```diff
@@ -176,11 +176,10 @@ def __init__(self, controller):
         self._create_discover_result = None
         self._convert_discover_result = None
         self._publish_discover_result = None
-        self._plugin_data = []
-        self._plugin_data_with_plugin = []
 
-        self._stored_plugins = set()
-        self._current_plugin_data = []
+        self._plugin_data_by_plugin_id = {}
```

Comment:
This could also be shortened to this if that's preferred - let me know.
```suggestion
        self._plugin_data_by_id = {}
```